graphics/tinygl/zdirtyrect.h

```diff
@@ -156,7 +156,7 @@ class BlittingDrawCall : public DrawCall {
 	};
 
 	BlittingDrawCall(BlitImage *image, const BlitTransform &transform, BlittingMode blittingMode);
-	virtual ~BlittingDrawCall() { }
+	~BlittingDrawCall();
```

Comment:
Thanks for keeping an eye out for C++ mistakes. For some reason I remembered "virtual" methods as being left unimplemented, but RTFM'ing a bit I see that's not what this keyword is about. Fixed.